test_dir_1/helloworld.py

```diff
@@ -0,0 +1,2 @@
+print "helloworld"
```

Comment:
A commit was made on test_dir_1/helloworld.py in RiflerRick/BB_MCS_Test authored by @RiflerRick